sessions/slides/introduction-to-nowcasting.qmd

```diff
@@ -56,19 +56,22 @@ Remember from [biases in delay estimation](../biases-in-delay-distributions):
 - reporting of events can be triggered by the **secondary** event
 - in that case, longer delays might be missing because whilst the *primary events* have occurred the *secondary events* **have not occurred yet**
 
-Nowcasting is exactly this if the seondary event is the data being available for analysis.
+Nowcasting is exactly this!.
```

Comment:
why not point to the specific issue? In the earlier session the secondary event was symptom onset so it might be confusing to say it's the same?

Reply:
I didn't really understand this point when reading it. I think maybe if it was worded: If the data is observed based on the secondary event?

Reply:
I thought:

## Session on convolutions
- primary event: infection
- secondary event: symptom onset

## Session on nowcasts
- primary event: symptom onset
- secondary event: report (or more precisely: onset enters the data, or: data is available for analysis)